### ISSUE TITLE
Broken links

### ISSUE THREAD

**Issue by sbliven** [SciCatProject/documentation#59]
I used [broken-link-checker](https://github.com/stevenvachon/broken-link-checker) to find broken links in the website. I used the command

```
npm install broken-link-checker
npx blc  -ro http://localhost:4040/ --exclude-external | grep BROKEN | sort -u
```

We should fix these.
- [ ] /Development/Development/Development_Methods.html from: /Development/v3.x/
- [ ] /Development/Development/Development_Methods.html from: /Development/v4.x/
- [ ] /Development/Operator from: /Development/v3.x/
- [ ] /Development/Operator from: /Development/v4.x/
- [ ] /Development/v3.x/Environment.md from: /Development/v3.x/OIDC.html
- [ ] /Development/v4.x/Documentation/v4.x/backend/configuration.md from: /Development/v4.x/overview.html
- [ ] /Development/v4.x/OIDC.html from: /Development/v4.x/Login.html
- [ ] /Development/v4.x/OIDC.md from: /Development/v4.x/Login.html
- [ ] /Development/v4.x/Theming.md from: /Development/v4.x/Login.html
- [ ] /Development/v4.x/Theming.md from: /Development/v4.x/frontend/configuration.html
- [ ] /Development/v4.x/frontend/testing.md from: /Development/v4.x/Migration.html
- [ ] /Development/v4.x/frontend/testing.md from: /Development/v4.x/backend/testing/user_authorization.html
- [ ] /Development/v4.x/testing.md from: /Development/v4.x/backend/testing_backend.html

**Comment by sbliven** [comment]
Some of these are straightforward typos and can be easily fixed. I wanted to ask about a couple though.

1. @LAShemilt From Development/v3.x/OIDC.md:
> `These setting are accomplished by modifying the frontend [environment document](./Environment.md).`
I don't think the frontend uses a `.env` file. I guess this should mean `config.json` and the link go to `./Configuration.md`?

2. What's the plan with the `Development/v4.x/old` directory? OIDC.md and Theming.md are in that folder but linked one level up. Do the 'old' files need to get updated and promoted to the correct place in the hierarchy?

**Comment by sbliven** [comment]
While documenting #61 I noticed `Development/v4.x/backend/configuration_oidc.md`. Should links to `v4.x/OIDC.md` point to this?

**Comment by LAShemilt** [comment]
Yes you are correct  Development/v3.x/OIDC.md should point to `[./Configuration.md]` and not `[./Environment.md]` . I also agree the text should say "configuration file" rather "environment file"
